contracts/interfaces/WETH9.sol

```diff
@@ -8,5 +8,5 @@ interface WETH9 {
 
     function balanceOf(address guy) external view returns (uint256 wad);
 
-    function transfer(address guy, uint256 wad) external;
+    function transfer(address guy, uint256 wad) external returns (bool);
```

Comment:
Didn't even realize we defined the interface incorrectly until I tried to use this in a Soldidity test